tests/integration/test_cluster_failover.py

```diff
@@ -94,7 +94,7 @@ def test_put_entity_type_to_crashed_primary_replica(self):
                         lsof = subprocess.check_output(["lsof", "-i", ":%s" % port])
                     except subprocess.CalledProcessError:
                         pass
-                    sleep(0.5)
+                    sleep(1)
```

Comment:
It seems to fail intermittently without this sleep, perhaps due to getting an unexpected error as the Cluster node is in the process of starting up.

Reply:
do you still have the error? can you paste them here?